src/decorators.ts

```diff
@@ -100,6 +105,29 @@ export function observe(targets: string|string[]) {
   }
 }
 
+/**
+ * A TypeScript accessor decorator factory that causes the decorated accessor to
+ * be called when a property changes. `targets` is either a single property
+ * name, or a list of property names.
+ *
+ * This function must be invoked to return a decorator.
+ */
+export function computed<T = any>(...targets: (keyof T)[]) {
+  return (proto: any, propName: string, descriptor: PropertyDescriptor): void => {
+    const targetString = targets.join(',');
+    const propNameCased = propName[0].toUpperCase() + propName.slice(1);
```

Comment:
I don't think it's really necessary to name-case the property. Just add a separator before it below, like `_`.